### PR TITLE
Update dennis to 0.9

### PR DESCRIPTION
**Pull request by Trojan295**
Dennis library was updated to 0.9 and it contains the fix for https://github.com/willkg/dennis/issues/91.

This PR reverts https://github.com/coala/coala-bears/pull/1282 and updates dennis to 0.9 to solve #1284

### CODE REVIEW

File: bear-requirements.txt [hunk 1]
```diff
@@ -4,11 +4,10 @@ apertium-lint~=0.29
 autoflake~=0.7
 autopep8~=1.2
 bandit~=1.2
-click==6.6
 cmakelint~=1.3
 cppclean~=0.12.0
 cpplint~=1.3
-dennis~=0.8
+dennis~=0.9
```
Comment (RaiVaibhav) [inline bear-requirements.txt]: I think `~=` means `not equal to` and as per issue you have to update it to `version =0.9`, I think have to make it to `dennis==0.9` instead `dennis~=0.9`

Comment (RaiVaibhav) [reply bear-requirements.txt]: unack 996b0b2570a6c498fda2a30aa05beca5c8b19c44

Comment (jayvdb) [reply bear-requirements.txt]: No, we avoid `==` if possible.

Comment (jayvdb) [reply bear-requirements.txt]: ack 996b0b2

**Comment by jayvdb** [comment]
ack 996b0b2

**Comment by RaiVaibhav** [comment]
Problem in coverage on `appveyor` it's not 100% :confused:

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
> Problem in coverage on appveyor it's not 100% 😕

ftr, there was no problem here.

**Comment by jayvdb** [comment]
@rultor merge

**Comment by Makman2** [comment]
@rultor merge

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
temporary build problem, PR is still approved.

**Comment by RaiVaibhav** [comment]
ack 30c55f8971609b5003421fb1745987a6270f3f66

**Comment by Makman2** [comment]
hmm that's bad... I wanted to make this PR appear in the approved-list... anyway, keeping the tab open so I won't forget

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Pull request merged**